BigProducts/Simulation/BuildFile.xml

```diff
@@ -31,7 +31,8 @@
 <use name="Validation/HcalHits"/>
 <use name="Validation/HGCalValidation"/>
 <use name="geant4static"/>
-<flags DROP_DEP="geant4core"/>
+<use name="g4hepemstatic"/>
+<flags DROP_DEP="g%core"/>
```

Comment:
@hahnjo , though this will work but this can also drop dependency on any tool which matches `g%core`. I would suggest to explicitly drop `geant4core and g4hepemcore` i.e.
```
<flags DROP_DEP="geant4core"/>
<flags DROP_DEP="g4hepemcore"/>
```

Reply:
Ah yes, I forgot to mention: IIRC this doesn't work because the build system only looks for one `DROP_DEP`. Specifying
```
<flags DROP_DEP="geant4core g4hepemcore"/>
```
doesn't work either because of how the underlying make rules are written, which expect at most one white-space separated argument...

Reply:
@smuzaffar does this make sense, did I miss an easy option to make this work? If not, I don't have the time right now to look into expanding CMSSW's build system, and we would really like to make progress with trying G4HepEm. Is the current wild-card approach acceptable for the time being?

Reply:
@hahnjo , I thought `<flags DROP_DEP="geant4core g4hepemcore"/>` should work, let me check.

Reply:
@hahnjo , https://github.com/cms-sw/cmsdist/pull/8170 should allow multiple `DROP_DEP` packages . So 
```
<flags DROP_DEP="geant4core g4hepemcore"/>
```
OR
```
<flags DROP_DEP="geant4core"/>
<flags DROP_DEP="g4hepemcore"/>
```

should both work once we have IB with cms-sw/cmsdist#8170

Reply:
Very cool! I will update this PR once there is an IB and I can test :smiley:

Reply:
@hahnjo  https://github.com/cms-sw/cmsdist/pull/8170 is merged and available in 12.6.X Ibs

Reply:
@smuzaffar thanks, worked perfectly as expected now! I only tested with `el8_amd64_gcc11` because I couldn't find the latest IB with `gcc10` - is `gcc11` now the default going forward?